### PR TITLE
Publish peripheral data as well, even if we don't use it ourselves

### PR DESCRIPTION
**Pull request by muffato**
TOLIT-2200

This PR is to _publish_ more output files from the pipeline. Those things were not originally included because we don't need them ourselves (we'll get the files via other pipelines) but someone running the pipeline may find those interesting.

At the same time, I found that the full-test didn't pass because of some recent changes I did.

Keeping it as a draft for now until the base branch is merged. I also need to document the changes in `docs/output.md`

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by muffato** [comment]
It's ready for review

File: docs/output.md [hunk 1]
```diff
@@ -53,12 +56,56 @@ BUSCO results generated by the pipeline (all BUSCO lineages that match the claas
...
+    - `full_table.tsv`: Coordinates of the annotated BUSCO genes as a tab-separated file.
+    - `missing_busco_list.tsv`: List of the BUSCO genes that could not be found.
+    - `*_busco_sequences.tar.gz`: Sequences of the annotated BUSCO genes. 1 _tar_ archive for each of the three annotation levels (`single_copy`, `multi_copy`, `fragmented`), with 1 file per gene.
+    - `hmmer_output.tar.gz`: Archive of the HMMER alignment scores.
+
+</details>
+
+### Read alignments
+
+Read alignments in BAM format -- only if the pipeline is run with `--align true`.
```
Comment (gq1) [inline docs/output.md:73]: Can we remove `true` from here?

Comment (muffato) [reply docs/output.md:73]: Probably. Let me check

**Comment by muffato** [comment]
> I can see `master` still being used in this ci? https://github.com/sanger-tol/blobtoolkit/blob/dev/.github/workflows/branch.yml
> 
> if you haven't changed anywhere else, I can do a quick PR.

Yes, please

File: docs/output.md (hunk 1 above)
Comment (gq1) [reply docs/output.md:73]: I always do for boolean command line options without `true`.

I also noticed no any tests with this option to test the alignment subworkflow. Maybe we can add it later.

Comment (muffato) [reply docs/output.md:73]: There's a profile named `test_raw` to test the alignment. That's what I used locally. I don't remember how much longer it takes though

File: docs/output.md [hunk 2]
```diff
@@ -26,8 +29,8 @@ The files in the BlobDir dataset which is used to create the online interactive
 <summary>Output files</summary>
 
 - `blobtoolkit/`
-  - `<accession>/`
-    - `*.json.gz`: files generated from genome and alignment coverage statistics
+  - `<assembly-name>/`
+    - `*.json.gz`: files generated from genome and alignment coverage statistics.
```
Comment (gq1) [inline docs/output.md:33]: I only can see these output when running `test_raw` profile, optional or not?

**Comment by gq1** [comment]
I have successfully run both `test` and `test_raw` profile on the farm as a new users.

Except the error with the two newly pulled singularity images:
`docker.io-pacificbiosciences-pyyaml-5.3.1.img`
`docker.io-genomehubs-blobtk-0.5.1.img`

And the error message:
```
Command error:
  FATAL:   container creation failed: mount /proc/self/fd/12->/software/singularity/3.11.4/var/singularity/mnt/session/rootfs error: while mounting image /proc/self/fd/12: kernel reported a bad superblock for squashfs image partition, possible causes are that your kernel doesn't support the compression algorithm or the image is corrupted
```
I have tried different version of singualrity on the new farm, all have the same problem. This is probably caused by a combination of farm22 and docker image?

File: docs/output.md (hunk 2 above)
Comment (muffato) [reply docs/output.md:33]: These should be **always** there. Can you show me the output directory ?

Comment (gq1) [reply docs/output.md:33]: It is OK now. Probably my previous run failed somehow.

File: docs/output.md (hunk 1 above)
Comment (muffato) [inline docs/output.md:73]:
```suggestion
Read alignments in BAM format -- only if the pipeline is run with `--align`.
```

**Comment by muffato** [comment]
Since #97 is taking a while, I've rebased this branch onto `dev`

**Pull request merged**